sql/parse/parse.go

```diff
@@ -1372,6 +1409,22 @@ func convertClose(ctx *sql.Context, closeCursor *sqlparser.CloseCursor) (sql.Nod
 	return plan.NewClose(closeCursor.Name), nil
 }
 
+func convertLoop(ctx *sql.Context, loop *sqlparser.Loop, query string) (sql.Node, error) {
```

Comment:
Parse tests for these?

Reply:
Just one or two parse tests for all the features in stored procs is probably plenty